install-components.md

```diff
@@ -1119,12 +1119,69 @@ To install Tanzu Build Service using the Tanzu CLI:
     >If the command times out, periodically run the installation verification step provided in the
     >following optional step. Image relocation continues in the background.
 
-1. (Optional) Verify the clusterbuilders created by the Tanzu Build Service install by running:
+### <a id='tbs-tcli-install-offline'></a> Install Tanzu Build Service using the Tanzu CLI (Air-Gapped)
+
+Tanzu Build Service can be installed to a Kubernetes Cluster and registry that are air-gapped from external traffic.
+
+These steps assume that you have installed the TAP packages in your air-gapped environment.
+
+### <a id='tbs-tcli-install-tbs-offline'></a> Install the TBS package (Air-Gapped)
+
+1. Gather the values schema by running:
+
+    ```
+    tanzu package available get buildservice.tanzu.vmware.com/1.4.2 --values-schema --namespace tap-install
+    ```
+
+2. Create a `tbs-values.yaml` file. Below are the required fields for an air-gapped installation.
+
+    ```
+    ---
+    kp_default_repository: REPOSITORY
+    kp_default_repository_username: REGISTRY-USERNAME
+    kp_default_repository_password: REGISTRY-PASSWORD
+    pull_from_kp_default_repo: true
```

Comment:
`pull_from_kp_default_repo` is interesting (especially within airgapped tap). Because I would probably still expect the user to use secret gen controller exported secret for pull.

Reply:
oh yea that's true